modules/external-secret-store/README.md

```diff
@@ -0,0 +1,63 @@
+# How to use
+
```

Comment:
this module also seems tightly related to aws and here it will be good to have its name contain "aws" word ?